src/entities.py

```diff
@@ -150,21 +150,34 @@ class Ingredients:
     """A dictionary of all ingredients (from the Studentenwerk) with their description."""
 
     fmi_ingredient_lookup = {
-        "Gluten": "Gl",
-        "Laktose": "Mi",
-        "Milcheiweiß": "Mi",
-        "Milch": "Mi",
-        "Ei": "Ei",
-        "Hühnerei": "Ei",
-        "Soja": "So",
-        "Nüsse": "Sc",
-        "Erdnuss": "En",
-        "Sellerie": "Sl",
-        "Fisch": "Si",
-        "Krebstiere": "Kr",
-        "Weichtiere": "Wt",
-        "Sesam": "Se",
-        "Senf": "Sf",
+        "a": "Gluten",
+        "aW": "Weizen",
+        "aR": "Roggen",
+        "aG": "Gerste",
+        "aH": "Hafer",
+        "aD": "Dinkel",
+        "aHy": "Hybridstämme",
+        "b": "Krebstiere",
+        "c": "Eier",
+        "d": "Fisch",
+        "e": "Erdnüsse",
+        "f": "Soja",
+        "g": "Milch",
+        "u": "Laktose",
+        "h": "Schalenfrüchte",
+        "hMn": "Mandeln",
```

Comment:
Looks like there are new ingredients. Could you please update the list in the `README.md`?

Reply:
They are from the FMI bistro. But below the _CLICK ME_, it says "based on Studentenwerk München". So it seems that the ingredients in the README do not contain the ingredients from the FMI bistro.

Reply:
Before, the ingredients were mapped to the same symbols as in the Studentenwerk menus (e.g. Soja -> So, ), I think this is what you also meant with your proposal for the enums in #48.

Here, you now create new labels/symbols (Soja, Milch, ...), I think it would be better, to reuse the existing ones, as they effectively mean the same thing. For example "f": "So", "c": "Ei", "hMn": "ScM", ...

Anyway, I think we should not use the german words here, as we keep everything else in english.

Reply:
The README `CLICK ME` should contain all ingredients we are aware of. The text "based on Studentenwerk München" since it already contains the ingredients of the "Mediziner Mensa".

I agree. First of all we should use existing ingredients before creating new ones. For this PR I think it's fine to have German ingredients to keep them the same as all others.

Reply:
@Philipp000: I agree. But I am already doing this in #60. So I will set this PR to WIP. Then it can be merged with #60.

Reply:
@COM8: I think, a good idea is to automatically create a JSON from the `Ingredient` enum I am introducing in #60. Do you think, we can then replace the `CLICK ME` with a link to the JSON? This way, we know that the README, enum and JSON file are always in sync.

Reply:
Yes, this would be a preferred version since then we do not have to maintain it in two places.

Reply:
Since master should be production ready, please either update the README, we wait until #60 is ready as well.